test/distributed/fsdp/test_fsdp_ignored_modules.py

```diff
@@ -6,6 +6,7 @@
 import torch.nn as nn
 from torch import distributed as dist
 from torch.distributed.fsdp import FullyShardedDataParallel as FSDP
+from torch.distributed.fsdp.api import ShardingStrategy
```

Comment:
nit: Let us just import from `.fsdp` for now even though `.fsdp.api` works.
```suggestion
from torch.distributed.fsdp import ShardingStrategy
```